riptide-spring-boot-autoconfigure/src/main/java/org/zalando/riptide/autoconfigure/FailsafePluginFactory.java

```diff
@@ -52,41 +59,62 @@ public static Plugin createCircuitBreakerPlugin(
                 .withDecorator(composite(decorators));
     }
 
+    //TODO: add more properties in config for more flexible CircuitBreaker configuration
     public static CircuitBreaker<ClientHttpResponse> createCircuitBreaker(
             final Client client,
             final CircuitBreakerListener listener) {
 
-        final CircuitBreaker<ClientHttpResponse> breaker = new CircuitBreaker<>();
+        final CircuitBreakerBuilder<ClientHttpResponse> breakerBuilder = CircuitBreaker.builder();
```

Comment:
Now `CircuitBreaker` and other policies are created via `builder()` method and can't be changed after - this limits how developers can modify beans after creation. 
We can think about supporting more Failsafe properties in our configuration

Reply:
I meant not new properties, but supporting more of existing properties. Earlier developers could autowire CircuitBreaker bean for example and add some logic like `handleIf(...)` - now CircuitBreaker and other policies can't be changed after creation. So we can support more of their properties in our autoconfiguration instead